### ISSUE TITLE
Composition: recipient becomes '+1' needlessly

### ISSUE THREAD

**Issue by LinuxOnTheDesktop** [thunderbird/thunderbird-android#6087]
### Checklist

- [X] I have used the search function to see if someone else has already submitted the same bug report.
- [X] I will describe the problem with as much detail as possible.

### App version

6.000

### Where did you get the app from?

Google Play

### Android version

12

### Device model

Asus Zenfone

### Steps to reproduce

1) Start to compose an email to a recipient address of 22 characters. The whole address, even when 'pilled', fits on my screen.

2) Move caret to the main field i e. the field for the text of the email. So doing makes the recipient's address appears only as '+1'.

### Expected behavior

The user should be able to see the recipient's address, at least when there is only a single recipient and/or there is no need to truncate any address. For, that lets the user see what she is doing.

### Actual behavior

The recipient's address appears only as

> +1

(within the 'To') field. The address does get displayed when I click on the 'to' field but there was never a need for truncation.

**Comment by hamishmb** [comment]
Yep, I also have this, it's annoying.

**Comment by hamishmb** [comment]
This stops you from viewing or changing the addresses, so I think it's quite a bad bug, given there's no apparent way to expand the addresses again.

**Comment by a-name-not-taken** [comment]
Agreed. Very dangerous to have the recipient essentially hidden especially when it may not be what is expected.

**Comment by LinuxOnTheDesktop** [comment]
I note:

- it is _coming up for a year and a half_ since I filed this bug;
- comments above show that two other people agree with me that it is a _serious_ bug;
- seemingly the bug report has received _no developer attention_.
